eng/pipelines/templates/pipeline.yml

```diff
@@ -223,7 +223,7 @@ stages:
       MSBuildEnableWorkloadResolver: false
     condition: "and(succeeded(), ne(variables['RunFunctionalTestsOnWindows'], 'false'))"
     pool:
-      name: VSEngSS-MicroBuild2022-1ES
+      name: VSEngSS-MicroBuild2022Preview-1ES
```

Comment:
The pool changed is required, I'm guessing, because you want the daily builds now, or...?

Reply:
.NET SDKs have a "minimum MSBuild version" configuration, and will error out when the detected version of msbuild is lower. It's also why when you go to https://dotnet.microsoft.com/en-us/download/dotnet/7.0, on the 8th of August there were 3 differnt .NET 7 SDKs installed. 7.0.400 has a minimum requirement of VS/MSbuild 17.7, 7.0.307 has a minimum requirement of 17.6, etc.

Anyway, the .NET 8 SDK has a minimum requirement of msbuild 17.8, and therefore would error out when we try to build with `msbuild` (rather than `dotnet build`).

[As Nikolche wrote](https://github.com/NuGet/NuGet.Client/pull/5402#discussion_r1317461185), this is one of the problems with using the same SDK to both build our product, as well as use for Dotnet.Integration.Tests